cli/cli-client/src/main/java/org/infinispan/cli/CLI.java

```diff
@@ -270,8 +282,6 @@ private void help(PrintStream out) {
       out.printf("Usage: cli [OPTION]...\n");
       out.printf("  -c, --connect=URL         %s\n", MSG.cliHelpConnect(Version.getBrandName()));
       out.printf("                            %s\n", MSG.cliHelpConnectHTTP());
-      out.printf("                            %s\n", MSG.cliHelpConnectJMXRMI());
-      out.printf("                            %s\n", MSG.cliHelpConnectJMXRemoting());
       out.printf("  -f, --file=FILE           %s\n", MSG.cliHelpFile());
```

Comment:
"If FILE is '-', then commands will be read from stdin."
I was unable to make it work. is it supported?